uni/lib/remorse.icn

```diff
@@ -0,0 +1,157 @@
+############################################################################
+#
+#	File:	  remorse.icn
+#
+#	Subject:  Procedures to convert a string to and from Morse code
+#
+#	Author:	  Don Ward
+#
+#	Date:	  October 2022
+#
+############################################################################
+#
+# This is a re-implementation of morse.icn which, compared to its predecessor:
+#	  handles more punctuation characters,
+#	  has a reverse mapping from Morse to text,
+#	  fails if a character that cannot be encoded is encountered,
+#	  is about twice as fast,
+#	  is easier to understand and extend.
+#
+############################################################################
+
+package remorse
+
+# ----------------------------------------------------------------------
+# translate s into Morse code
+# For compatibility with morse(), words are separated by five spaces.
+procedure Morse(s)
```

Comment:
It is not common to capitalize library function names. Since this is in a package now, we can just do morse().

Reply:
Modified as requested. Morse() was originally capitalised as it's the name of a person (Samuel Morse) but I'm happy with morse().